rest/blip_sync.go

```diff
@@ -124,30 +132,36 @@ func (ctx *blipSyncContext) register(profile string, handlerFn func(*blipHandler
 
 	ctx.blipContext.HandlerForProfile[profile] = func(rq *blip.Message) {
 
-		ctx.LogTo("Sync", "%s %q ... %s", rq, profile, ctx.effectiveUsername)
+		startTime := time.Now()
 
 		db, _ := db.GetDatabase(ctx.dbc, ctx.user)
 		handler := blipHandler{
 			blipSyncContext: ctx,
 			db:              db,
 		}
 
+		ctx.incrementSerialNumber()
```

Comment:
I changed it so that the blipSyncContext holds the serial number counter, and whenever a new blipHandler is created it will increment the counter and store a snapshot of the counter in the blipHandler, which it will use for it's logging.

Example logs: https://gist.github.com/tleyden/8d75c0fba4cf5aaba025aad48bbbc92b

There are two unique blipSyncContexts:

* [2c49c010]
* [7b6b2a1f]